storybookjs/storybook
# UI: Do not display menu toggle when `singleStory=true`

Issue: #15553

## What I did

I added a condition on `menuTool` not to display if singleStory is set

## How to test

- open a story and append `&singleStory=true` to the URL
- the menu toggle should not be displayed
